zcash_test_vectors/zip_0244.py

```diff
@@ -351,9 +362,15 @@ def randbytes(l):
     consensusBranchId = 0xc2d6d0b4 # NU5
 
     test_vectors = []
-    for _ in range(10):
-        tx = TransactionV5(rand, consensusBranchId)
-        txid = txid_digest(tx)
+    for i in range(10):
```

Comment:
We need separate set of test vectors for V5 and for V6, not to be mixed.

Reply:
Removed V6 test vectors from here, will do it separately in the next PR.